src/i18n/en.json

```diff
@@ -191,6 +192,7 @@
         "request-type": "What kind of request do you want to generate?",
         "access-request": "Access request",
         "erasure-request": "Erasure request",
+        "objection-request": "Objection against direct marketing",
```

Comment:
This is pretty long and looks ugly on mobile devices. What about "Direct marketing objection"?

Reply:
Feels like a bit of a "German-ism" to me but sure.